runtime/testy/test_outputs/hibikeSubscribeDevice_output

```diff
@@ -1,14 +1,14 @@
-0
 1
 2
+3
```

Comment:
I'm really confused by this. The old output must have been generated by actually running the code... as has this new output. But what has changed?

This points to some subtle non-determinism in the runtime. Best-case scenario is that the test-case framework is subtly flawed (@yizow: you should remember that I warned about non-determinism when this particular style of test framework was introduced). The worst-case scenario is that there is some subtle timing bug lurking in runtime.